Documentation/grpc-metadata.md

```diff
@@ -194,7 +199,8 @@ func (s *server) SomeStreamingRPC(stream pb.Service_SomeStreamingRPCServer) erro
 
 #### Unary call
 
-To send header and trailer to client in unary call, the server can call [SendHeader](https://godoc.org/google.golang.org/grpc#SendHeader) and [SetTrailer](https://godoc.org/google.golang.org/grpc#SetTrailer) functions in module [grpc](https://godoc.org/google.golang.org/grpc).
+To send header and trailer to client in unary call, the server can call
+[SendHeader][2] and [SetTrailer][3] functions in module [grpc][6].
```

Comment:
Should we point people towards `SetHeader` rather than `SendHeader`? The former can be called multiple times (but doesn't immediately trigger sending them), which I think is what people want in most cases.

Ideally I think it would make sense to document why one would use one over the other, as (I myself don't have a clear understanding of when SendHeader would be useful).

Reply:
Yes, I agree that we should point people towards `SetHeader` here. We don't even have to mention `SendHeader` here, and in the API doc, we can clarify the difference between the two and when to use the latter. Wdyt @atollena ?

Reply:
Yes, that makes sense. Looking at the docs for SetHeader/SendHeader, it describes the contract very well but does not tell users when SendHeader might be what users want. Do you have an example of such a case?

Reply:
I have replaced to `SetHeader` in grpc-metadata.md`. Will create a separate PR for docs (grpc.io). One more thing, shall we show usage of `SetHeader` and `SetTrailer` in [metadata_interceptor example](https://github.com/grpc/grpc-go/blob/master/examples/features/metadata_interceptor/server/main.go#L46)?

Reply:
I added the `SetHeader` and `SetTrailer` in metadata_interceptor example. Let me know if it looks good